iothub/device/tests/ConnectionString/DeviceClientConnectionStringTests.cs

```diff
@@ -96,6 +96,15 @@ public void DeviceClientConnectionStringX509CertTest()
             var deviceClient = DeviceClient.CreateFromConnectionString(connectionString);
         }
 
+        [TestMethod]
```

Comment:
As you are on this file, any chance you can take a look at the ignored tests? There are many and there is a TODO to reenable them. We should have good test coverage for this. Also I recently found it was tracked here - https://github.com/Azure/azure-iot-sdk-csharp/issues/583.

If you feel it is way out of the current scope, we can make a task for it. Let me know.

Reply:
Ended up abandoning this PR :)